src/main/java/org/isf/accounting/manager/BillBrowserManager.java

```diff
@@ -53,9 +53,10 @@ public class BillBrowserManager {
 	 * @param billPayments
 	 * @throws OHDataValidationException
 	 */
-	protected void validateBill(Bill bill, List<BillItems> billItems, List<BillPayments> billPayments) throws OHDataValidationException {
-        List<OHExceptionMessage> errors = new ArrayList<>();
 
+
```

Comment:
Why are you including files that exist in another PR?  (see https://github.com/informatici/openhospital-core/pull/1175/files).   And none of the changes requested made in that PR are included here.

Reply:
I'll resolve it from my side.